src/styles/_common/_typography.scss

```diff
@@ -86,3 +86,16 @@ $small: em(9px);
     @include font("IntervalNextReg");
     color: color("input", "label");
 }
+
+@mixin input-placeholder() {
+    ::-webkit-input-placeholder {@content}
+    :-moz-placeholder {@content}
```

Comment:
Since Firefox supports the standard version its not necessary 
https://caniuse.com/#search=placeholder